### PR TITLE
Generic image include

### PR DESCRIPTION
**Pull request by DrMagPie**
[preview](https://drmagpie.alt-f4.blog/ALTF4-10/)

combines:
- [X] figure.html
- [X] hires.html
- [X] imgur.html (kinda does we just need to use full link)
- Something missing?

TODO:
- [ ] Update Sample post

### CODE REVIEW

**Comment by justchen1369** [comment]
I can spend some time changing all of the old links to be imgur links.

File: _includes/image.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<p></p> <!-- Spacing -->
+<div class="image">
+  {% if include.href %}<a href="{{ include.href }}">{% endif %}
```
Comment (justchen1369) [inline _includes/image.html]: target should be _blank for the link.

Comment (DrMagPie) [reply _includes/image.html]: what do you mean? why?

**Comment by Conor12345** [comment]
Not sure I like the way the click to show high resolution shows up on top of the image. Would it be possible to have it below instead?

**Comment by DrMagPie** [comment]
Actually it's easier to align it to the top cuz caption on bottom can wrap
image
image


also this is an option
image
image
but on mobile it looks like shit

File: _sass/image.scss [hunk 2]
```diff
@@ -0,0 +1,28 @@
+.image {
+  position: relative;
+  text-align: center;
+  a {
+    text-decoration: none;
+    i {
+      position: absolute;
+      top: 50%;
```
Comment (DrMagPie) [inline _sass/image.scss:8]: @Conor12345 Fell free to play around with this parameters to align it the way you want.
`top:` and `bottom:`
ref: https://www.w3schools.com/howto/howto_css_image_text.asp

File: _includes/image.html (hunk 1 above)
Comment (justchen1369) [reply _includes/image.html]: This way clicking on the link will open it in a new tab.

Comment (Conor12345) [reply _includes/image.html]: Clicking needs to open a different, higher res version so the link needs to be explicitly given.

Comment (DrMagPie) [reply _includes/image.html]: mm makes sense

Comment (DrMagPie) [reply _includes/image.html]: @Conor12345 yes. `href` is high res version and `src` is version displayed on website

Comment (DrMagPie) [reply _includes/image.html]:
```
{% 
include image.html 
href='https://media.alt-f4.blog/ALTF4/13/base_new_map_og.jpg'  <!-- hires/original -->
src='https://media.alt-f4.blog/ALTF4/13/base_new_map.jpg' <!-- optimised -->
alt='Map view of rail network, showing only an eighth of the outposts.'
caption='Map view of rail network, showing just 27 of the 216 outposts.' 
%}
```

**Comment by Conor12345** [comment]
Could we permanently update the draft page to include these too?

**Comment by Conor12345** [comment]
Also is it possible for the image to open in a new tab, rather than the current one? Unless there is some reason it shouldn't?

**Comment by justchen1369** [comment]
That's what target _blank is for.

**Comment by Conor12345** [comment]
> That's what target _blank is for.

Yep my bad, was on test site which hadn't been updated.

**Pull request merged**